pages/docs/manual/latest/function.mdx

```diff
@@ -409,33 +409,36 @@ add(1, 2);
 
 </CodeTab>
 
-If you need to call a curried function with a single unit `()` argument, you can use the `ignore()` function:
```

Comment:
🗒️ Looks like this is no longer required for v10?

Reply:
Indeed, plus it should have said "uncurried".